src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -27,25 +27,42 @@
 from pybuilder.errors import BuildFailedException
 from pybuilder.utils import assert_can_execute
 from pybuilder.utils import execute_command
+from pybuilder import scaffolding as SCAFFODING
+from pybuilder.core import NAME_ATTRIBUTE
 
-__author__ = 'Thomas Prebble'
+__author__ = 'Thomas Prebble', 'Marcel Wolf'
 
 
 use_plugin("core")
 
 
-DEFAULT_SPHINX_BUILDER = "html"
-DEFAULT_SPHINX_SOURCE_DIR = ""
-DEFAULT_SPHINX_OUTPUT_DIR = "_build/"
+DEFAULT_SPHINX_OUTPUT_DIR = SCAFFODING.DEFAULT_DOCS_DIRECTORY + "/_build/"
```

Comment:
My mistake, it's not a problem at all since we pull the actual dir out of the project property.